airbyte-oauth/src/test/java/io/airbyte/oauth/flows/TikTokMarketingOAuthFlowTest.java

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
this is just a completely new java test. (it was missing before)

Reply:
good addition!